src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -118,8 +118,8 @@
     private int readTimeout;
     /** @deprecated Stored as a list of PodLabels */
     @Deprecated
-    private Map<String, String> labels;
-    private List<PodLabel> podLabels = new ArrayList<PodLabel>();
+    private transient Map<String, String> labels;
```

Comment:
@Vlatombe Thanks for catching that, I had it in there and removed it when I was testing.